app/controllers/tickets_controller.rb

```diff
@@ -30,6 +30,8 @@ def show
     @reply = @ticket.replies.new(user: current_user)
     @reply.set_default_notifications!
 
+    @private_message = @ticket.private_messages.new({user: current_user, ticket: @ticket})
```

Comment:
Line is too long. [90/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.